nodelocaldns/go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/go-logr/logr v0.1.0
 	golang.org/x/tools v0.0.0-20191010075000-0337d82405ff
 	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
-	k8s.io/api v0.18.2
-	k8s.io/apimachinery v0.18.2
-	k8s.io/client-go v0.18.2
+	k8s.io/api v0.18.4
+	k8s.io/apimachinery v0.18.4
+	k8s.io/client-go v0.18.4
 	k8s.io/klog v1.0.0
 	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200512162422-ce639cbf6d4c
```

Comment:
We should figure out when we want to create a version in kubebuilder-declarative-pattern :-)